### PR TITLE
Small changes in the current schema of the docker files. 

### PR DESCRIPTION
**Pull request by ypriverol**
Minor changes in the schema LABEL names to make them more consistent. Please @osallou @prvst @mr-c take a look at this PR. The basic idea here is to make standard the current names. We should consider a major redefinition of the schema for the future including the comments from @mr-c and other about been compatible with http://label-schema.org/rc1/ . I recommend to accept this PR first as small change to the schema and then move into a major changes. 

@mr-c I think is more important now that you check if the current entry-point for containers is fine for `cwl` and also the use of biodocker user `USER biodocker` is also fine. We also recommend the use of the working directory `WORKDIR /data/` please give us your feedback on that.

### CODE REVIEW

**Comment by osallou** [comment]
This would be fine for me but we need to coordinate. This will impact some scripts on continuous integration that checks some label rules as well as get container name and version from labels
However modifying specs multiple times is a bit dangerous as may break several tools (web site, indexation etc...), should be limited as much as possible.

**Comment by mr-c** [comment]
Ideally, from a CWL perspective, there would not be a `CMD` either so that users wouldn't be tempted to leave `basename` blank in the CWL descriptions (thus making them non-portable).

The `WORKDIR` does not matter as the CWL reference runner mounts in its own.

It is good that the `USER` isn't `root` to discourage Dockerfiles that depend on that; note that the CWL reference runner (and many others) runs the as the "external" user so that file permissions are reasonable.

**Comment by ypriverol** [comment]
@mr-c Thanks for your comment. @osallou a couple of things: 
1- we should avoid to put CMD in the Dockerfiles because this will give a default entry point. 

@mr-c are you happy to use @biodocker as the user?

**Comment by osallou** [comment]
for CMD, same as ENTRYPOINT. Setting it is interesting as user does not
have to know where tool is installed, if not in path. But as it is not
always the case for all containers, would make sense they all follow the
same way to work, ie no ENTRYPOINT/CMD. However, would help users to know
which *exe* are available in container. In debian Dockerfiles, I define a
LABEL for this:

    binaries="/usr/bin/abacas"

which may be a list of comma separated binaries (even if in this case
binary is in PATH)

File: container-specs.md [hunk 1]
```diff
@@ -7,16 +7,16 @@ We explain here the containers metadata, their meaning and use in BioContainers:
...
-| tags                  | Tags about the software that enable to find and classify the software tool.                                                | tags: proteomics, mass spectrometry, biocontainers       |
+|-----------------------|----------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------|
+| base_image            | The original image where the software has been built | base_image: biodckr/biodocker |
+| software.version      | Version of the software and tool                     | software.version: 2015020     |
+| software              | Name of the software                                 | software: Comet               |
+| about.summary         | A short description of the software or tool.         | about.summary: Peptide identification|
+| about.home            | The original software website where  and examples.   | about.home: http://comet-ms.sourceforge.net/  |
+| about.documentation   | URL(s) containing information about software            | aobut.documentation: http://comet-ms.sourceforge.net/     |
```
Comment (osallou) [inline container-specs.md]: there is typo here (aobut)

Comment (ypriverol) [reply container-specs.md]: Typo has been fixed. I have also remove the CMD as suggested by @mr-c

**Comment by osallou** [comment]
should we not specify here what fields in LABELS are mandatory, and which one are optional?
for example, about.documentation could be optional, if doc is in homepage (about.home)

**Comment by osallou** [comment]
looks good for me

**Pull request merged**